src/containers/PaymentMethodsPage/PaymentMethodsPage.js

```diff
@@ -119,6 +200,19 @@ export const PaymentMethodsPageComponent = props => {
                   <FormattedMessage id="PaymentMethodsPage.deletePaymentMethod" />
                 </div>
               </div>
+            ) : (
+              <PaymentMethodsForm
+                className={css.paymentForm}
+                formId="PaymentMethodsForm"
+                initialValues={initalValuesForStripePayment}
+                onSubmit={handleSubmit}
+                handleRemovePaymentMethod={handleRemovePaymentMethod}
+                hasDefaultPaymentMethod={hasDefaultPaymentMethod}
+                addPaymentMethodError={addPaymentMethodError}
+                deletePaymentMethodError={deletePaymentMethodError}
+                createStripeCustomerError={createStripeCustomerError}
+                handleCardSetupError={handleCardSetupError}
+              />
```

Comment:
You should also handle inProgress states (e.g. `addPaymentMethodInProgress`). Form should show spinner in the submit button if the saving is in progress.